### PR TITLE
Change name of line fade in RenderableTrail from Fade to LineFade

### LINKED ISSUE #2856: RenderableTrail has a property called Fade which shadows renderable property Fade
**Describe the bug**
RenderableTrail has a property called Fade (under Appearance) which shadows renderable property Fade. This leads to undefined behavior when using the URI for fade - it is unclear which fade is accessed. To fix - rename the fade property in RenderableTrails and everywhere it is used.

### PR DESCRIPTION
**Pull request by ylvaselling**
Close #2856

### CODE REVIEW

**Comment by WeirdRubberDuck** [comment]
Not sure that this change is necessary; Shouldn't we just be able to handle the checking against the property better in the UI?  Otherwise I'm pretty sure the error will emerge again at some point. 

Worth a discussion;  I know that there is some complication in doing this since not only renderables have the Fade property.

**Comment by WeirdRubberDuck** [comment]
Looking a bit more into detail of this property, renaming it to "LineFade", makes sense. I mean, it shows up as "Line Fade" in the UI anyways 

However, we should still address the underlying cause of the problem, which is the check in the UI code.  But I think it's ok to go ahead and merge this

File: modules/base/rendering/renderabletrail.cpp [hunk 1]
```diff
@@ -87,7 +87,7 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo FadeInfo = {
```
Comment (WeirdRubberDuck) [inline modules/base/rendering/renderabletrail.cpp:89]:
```suggestion
    constexpr openspace::properties::Property::PropertyInfo LineFadeInfo = {
```

File: modules/base/rendering/renderabletrail.cpp [hunk 2]
```diff
@@ -139,7 +139,7 @@ namespace {
         std::optional<bool> enableFade;
 
         // [[codegen::verbatim(FadeInfo.description)]]
```
Comment (WeirdRubberDuck) [inline modules/base/rendering/renderabletrail.cpp:141]:
```suggestion
        // [[codegen::verbatim(LineFadeInfo.description)]]
```

File: modules/base/rendering/renderabletrail.cpp (hunk 1 above)
Comment (WeirdRubberDuck) [reply modules/base/rendering/renderabletrail.cpp:89]: Also in the constructor of the Appearance property owner

**Review by alexanderbock — APPROVED** [review]

**Pull request merged**